package.json

```diff
@@ -6,16 +6,18 @@
   "scripts": {
     "browserify": "./node_modules/.bin/browserify lib/index.js --standalone blockstack -o ./dist/blockstack.js",
     "browserify-tests": "./node_modules/.bin/browserify lib/index.js --standalone blockstack -o ./tests/browserTests/bundle.js",
+    "uglify": "./node_modules/.bin/uglifyjs ./dist/blockstack.es5.js --compress --mangle reserved=['BigInteger','ECPair','Point'] --output ./dist/blockstack.min.js",
```

Comment:
Nice this is looking better. I would actually suggest using [terser](https://github.com/terser-js/terser) as uglify is not actively maintained anymore.

Reply:
alright, I'll give it a shot.